docs/user/guides/edit-source-links-sphinx.rst

```diff
@@ -8,7 +8,7 @@ You can use these variables in your own Sphinx theme as well.
 More information can be found on `Sphinx documentation`_.
 
 .. _`our Sphinx theme`: https://sphinx-rtd-theme.readthedocs.io/
-.. _`Sphinx documentation`: http://www.sphinx-doc.org/en/master/configuration.html#confval-html_context
+.. _`Sphinx documentation`: http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_context
```

Comment:
Thanks for your contribution @cagatay-y ! Would you rather replace this link by the appropriate intersphinx reference?

Reply:
I think those can't be used with intersphinx https://github.com/sphinx-doc/sphinx/issues/5562.